src/connector/src/sink/kafka.rs

```diff
@@ -220,10 +223,12 @@ impl KafkaSinkWriter {
         schema: Schema,
         pk_indices: Vec<usize>,
         is_append_only: bool,
+        identifier: String,
     ) -> Result<Self> {
         Ok(KafkaSinkWriter {
             config: config.clone(),
-            conductor: KafkaTransactionConductor::new(config).await?,
+            conductor: KafkaTransactionConductor::new(config, &identifier).await?,
```

Comment:
how about removing the abstraction? After decoupling, we can tolerant failure from network jitter, there is no need for a retry layer.

Reply:
LGTM. I think we can do this in another PR.